client/landing/stepper/declarative-flow/write.ts

```diff
@@ -73,9 +74,12 @@ const write: Flow = {
 			}
 		};
 
-		const goNext = () => {
+		const goNext = async () => {
 			switch ( _currentStep ) {
 				case 'launchpad':
+					if ( siteSlug ) {
+						await updateLaunchpadSettings( siteSlug, { launchpad_screen: 'skipped' } );
+					}
 					return window.location.assign( `/view/${ siteId ?? siteSlug }` );
```

Comment:
Should we not take the user to Customer Home? With `launchpad_screen` set to `skipped`, we won't redirect them to the fullscreen launchpad.
```suggestion
					return window.location.assign( `/home/${ siteSlug ?? siteId }` );
```